src/utils/convertObjectToMap.js

```diff
@@ -0,0 +1,27 @@
+const convertObjectToMap = (gridFSObject) => {
+  if (gridFSObject && typeof gridFSObject === "object") {
+    if (Array.isArray(gridFSObject)) {
+      return gridFSObject.map(convertObjectToMap);
+    }
+    const convertedObject = {};
```

Comment:
P1: 해당 부분도 개행 부탁드립니다.

```suggestion
    }
    
    const convertedObject = {};
```